Tests/test_Seq_objs.py

```diff
@@ -160,7 +143,6 @@ def pre_comp_function(x):
                             % (repr(example1), method_name, repr(example2), i, j)
                         )
                 except TypeError:
-                    # TODO - Check the alphabets do clash!
```

Comment:
Do we still get a TypeError? i.e. Can we remove the try/except now?

Reply:
Will do this separately. Git got confused.